ekncontent/ekncontent/eknc-xapian-bridge.c

```diff
@@ -110,9 +113,11 @@ eknc_xapian_bridge_finalize (GObject *object)
 {
   EkncXapianBridge *self = EKNC_XAPIAN_BRIDGE (object);
 
+  g_clear_object (&self->database_manager);
+  g_clear_object (&self->session);
```

Comment:
Can you remove the soup session (and possibly the libsoup dependency) entirely as well?

Reply:
Yes, the goal is to drop that in later commits.